resources/public/api/conf/1.0/application.raml

```diff
@@ -1,59 +0,0 @@
-#%RAML 1.0
----
```

Comment:
Please revert. This should not be deleted as the RAML and OAS need to match before deleting.